lib/repl.js

```diff
@@ -937,7 +937,6 @@ function complete(line, callback) {
     var flat = new ArrayStream();         // make a new "input" stream
     var magic = new REPLServer('', flat); // make a nested REPL
     replMap.set(magic, replMap.get(this));
-    magic.resetContext();
```

Comment:
`resetContext` have called in [constructor](https://github.com/nodejs/node/blob/master/lib/repl.js#L425),so it should be deleted.